index.bs

```diff
@@ -243,6 +243,50 @@ scenarios. Additional scenarios, including sample code, are given later in [[#sa
     * Web page shows that the selected user is signed in, and navigates to the signed-in page.
 
 
+### New device enrollment ### {#usecase-new-device-enrollment}
+
+This use case scenario illustrates how a [=[RP]=] can leverage a combination of a [=roaming authenticator=] (e.g., a USB security
+key fob) and a [=platform authenticator=] (e.g., a built-in fingerprint sensor) such that the user has:
+
+  - a "primary" [=authenticator=] that they use to authenticate on new-to-them devices (e.g., laptops, desktops) or on such
+    devices that lack a [=platform authenticator=], and
+  - a low-friction means to strongly re-authenticate on devices having [=platform authenticators=].
+
+Note: This approach of registering multiple [=authenticators=] for an account is also useful in account recovery use cases.
+
+Note: The various prompts and user-entered texts in this section are examples, and not normative.
+
+- First, on a desktop computer (lacking a [=platform authenticator=]):
+    * User navigates to `example.com` in a browser and signs in to an existing account using whatever method they have been using
+        (possibly a legacy method such as a password), or creates a new account.
+    * User navigates to account security settings and selects "Register security key".
+    * Website prompts the user to plug in a USB security key fob and enter a nickname for the key.
+    * User enters "Keychain key" and plugs in the USB security key.
```

Comment:
... and say only "User plugs in their USB security key." here.  another related comment is below.

Reply:
Yep.